tests/vmi_multus_test.go

```diff
@@ -521,16 +521,15 @@ var _ = Describe("[Serial]Multus", func() {
                     setenforce 0
                     ip link add ep1 type veth peer name ep2
                     ip addr add %s dev ep1
-	                ip addr add %s dev ep2
-	                ip addr add %s dev ep1
-	                ip addr add %s dev ep2
-                    mkdir -p /usr/local/bin
-                    curl %s > /usr/local/bin/qemu-ga
-                    chmod +x /usr/local/bin/qemu-ga
-		    curl %s > /lib64/libpixman-1.so.0
-                    systemd-run --unit=guestagent /usr/local/bin/qemu-ga
-                `, ep1Cidr, ep2Cidr, ep1CidrV6, ep2CidrV6, tests.GetUrl(tests.GuestAgentHttpUrl), tests.GetUrl(tests.PixmanUrl))
-				agentVMI := tests.NewRandomVMIWithEphemeralDiskAndUserdata(cd.ContainerDiskFor(cd.ContainerDiskFedora), userdata)
+                    ip addr add %s dev ep2
+                    ip addr add %s dev ep1
+                    ip addr add %s dev ep2
+                    sudo cp /home/fedora/qemu-guest-agent.service /lib/systemd/system/
```

Comment:
Where is this service file coming from? Do we really need a custom one, now that we have the agent properly installed?

Reply:
I explain it here. https://github.com/kubevirt/kubevirtci/pull/533

short answer is, guest agent can't be disabled (at least I couldn't practically make it work) without completely stopping it and removing the service file entirely from being loaded at boot. systemctl disable qemu-guest-agent doesn't do anything because there's a optional chain that forces guest agent on at boot. 

we have lots of logic all throughout the test suite that is using the presence of the guest agent connected condition to guarantee that the cloud-init logic has executed... so if the qemu guest agent starts at boot before cloud-init runs, then we get a wrong password prompt and other configuration won't be applied.  If we start the guest agent in cloud-init as the last step, then the timing works correctly. 

I copied the original service account to this folder where it can be re-installed. Automation does this in our fedora tools build step in kubevirtci, so it will continue to work as we adopt new versions of fedora